MekHQ/src/mekhq/campaign/mission/ScenarioObjective.java

```diff
@@ -323,18 +323,70 @@ public String toShortString() {
         case ForceWithdraw:
         case Capture:
         case Preserve:
-            return String.format("%s %s%s", getObjectiveCriterion().toString(), amountString, timeLimitString);
+            return String.format("<html>%s %s%s<span color='black'>%s%s</span></html>", getObjectiveCriterion().toString(), amountString, 
+                    timeLimitString, buildEffects(true), buildEffects(false));
         case ReachMapEdge:
-            return String.format("Reach %s edge with %s%s", edgeString, amountString, timeLimitString);
+            return String.format("<html>Reach %s edge with %s%s<span color='black'>%s%s</span></html>", edgeString, amountString, 
+                    timeLimitString, buildEffects(true), buildEffects(false));
         case PreventReachMapEdge:
-            return String.format("Prevent %s from reaching %s%s", amountString, edgeString, timeLimitString);
+            return String.format("<html>Prevent %s from reaching %s%s<span color='black'>%s%s</span></html>", amountString, edgeString, 
+                    timeLimitString, buildEffects(true), buildEffects(false));
         case Custom:
-            return String.format("%s%s%s", getDescription(), amountString, timeLimitString);
+            return String.format("<html>%s%s%s<span color='black'>%s%s</span></html>", getDescription(), amountString, 
+                    timeLimitString, buildEffects(true), buildEffects(false));
         default:
                 return "?";
         }
     }
     
+    private String buildEffects(boolean success) {
+        StringBuilder result = new StringBuilder();
+        List<ObjectiveEffect> effectCollection = success ? getSuccessEffects() : getFailureEffects();
+        
+        if (!effectCollection.isEmpty()) {
+            result.append("<br/>");
+        }
+        
+        for (ObjectiveEffect effect : effectCollection) {
+            boolean scaledEffect = effect.effectScaling == EffectScalingType.Linear ||
+                    effect.effectScaling == EffectScalingType.Inverted;
```

Comment:
Brackets, although implementing boolean comparisons (EffectScalingType::isLinear, EffectScalingType::Inverted) would be easier to read code. However, I'd say leave it as is for now and handle it in the future.